src/Tribe/Tickets.php

```diff
@@ -1514,12 +1537,13 @@ public static function get_event_attendees_count( $post_id, $args = [] ) {
 		 * @return array
 		 */
 		public static function get_all_event_tickets( $post_id ) {
-			$cache_key = self::$cache_key_prefix . $post_id;
-			$cache = new Tribe__Cache();
-			$tickets = $cache->get( $cache_key );
 
-			if ( is_array( $tickets ) ) {
-				return $tickets;
+			/** @var Tribe__Cache $cache */
+			$cache   = tribe( 'cache' );
+			$key     = __METHOD__ . '-' . $post_id;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 5 spaces

<sub>hash: 3a5f5021fb47f5b1bbed8712e486d588</sub>